src/hooks/useDisplayedWallets.ts

```diff
@@ -24,15 +27,21 @@ export const useDisplayedWallets = (): WalletInfo[] => {
   const injectedWallets = useMipdInjectedWallets();
 
   return useMemo(() => {
-    const phantomDetected =
-      injectedWallets.findIndex((wallet) => wallet.detail.info.rdns === PHANTOM_MIPD_RDNS) !== -1;
+    const phantomDetected = Boolean(window.phantom?.solana);
+    const keplrDetected = Boolean(window.keplr);
 
     const okxDetected =
       injectedWallets.findIndex((wallet) => wallet.detail.info.rdns === OKX_MIPD_RDNS) !== -1;
 
+    const injectedMetaMask = injectedWallets.find(
+      (wallet) => wallet.detail.info.rdns === METAMASK_MIPD_RDNS
+    );
+
     const enabledInjectedWallets = injectedWallets
       .filter(
         (wallet) =>
+          // Remove Metamask. We will show it no matter what at the front of
```

Comment:
fix grammar? 😆

Reply:
lmao, this comment got mangled 😆 what happened to the second line?!